checkpoint_quantum_firewall/README.md

```diff
@@ -0,0 +1,132 @@
+## Overview
+
+Check Point Next Generation Firewall is a security gateway that includes application control and IPS protection, with integrated management of security events. Additional features include Identity Awareness, URL Filtering, Anti-Bot, Anti-Virus, and Anti-Spam.
+
+## Setup
+
+### Installation
+
+To install the Checkpoint Quantum Firewall integration, follow the steps below:
+
+1. [Install][5] the 1.0 release (`checkpoint_quantum_firewall==1.0.0`).
```

Comment:
Could you add a section saying that this integration is shipped with the Agent by default with 7.52+? See https://github.com/DataDog/integrations-core/tree/master/tenable

Reply:
Done